packages/typespec-test/test/unionwidget_dpg/generated/typespec-ts/src/models/models.ts

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export interface Widget {
+  readonly id: string;
+  weight: number;
+  color: ColorType;
+}
+
+/** Type of ColorType */
+/** */
+export type ColorType = "red" | "blue";
+
+export interface Widget1 {
+  data: WidgetData0 | WidgetData1;
```

Comment:
I notice the definition part it is extened from Widget, why is there no this info generated?

```
model Widget {
  @key id: string;
  weight: int32;
  color: "red" | "blue";
}

model Widget1 extends Widget {
  data: WidgetData0 | WidgetData1
}
```

Reply:
will look into this tomorrow.

Reply:
fixed.